src/styling-panel-definition.js

```diff
@@ -11,7 +44,406 @@ const getStylingPanelDefinition = (bkgOptionsEnabled) => ({
       ref: 'components',
       useGeneral: true,
       useBackground: bkgOptionsEnabled,
+      items: {
+        labelSection: {
+          translation: 'Label',
+          component: 'panel-section',
+          show(data, handler, args) {
```

Comment:
What is this show function supposed to do? It just return true in the end, but there is a lot of logic in here which should not be here